src/test/java/org/opentripplanner/routing/fares/ATLFareServiceTest.java

```diff
@@ -140,7 +145,23 @@ public void tooManyRides() {
         );
         calculateFare(rides, Fare.FareType.electronicRegular, DEFAULT_RIDE_PRICE_IN_CENTS * 2);
 
-        // TODO: Any agencies with 3 transfer limit? GCT?
+        rides = Arrays.asList(
+            getRide(GCT_AGENCY_ID, 0),
+            getRide(MARTA_AGENCY_ID, 1),
+            getRide(MARTA_AGENCY_ID, 2),
+            // new transfer - only got 3 from GCT
+            getRide(MARTA_AGENCY_ID, 3)
```

Comment:
I think you need to add one more ride. I see four rides and three transfers in this list, which under GCT should be on the same ticket.